### PR TITLE
v2 

### PR DESCRIPTION
**Pull request by scarlehoff**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by scarlehoff** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
   version: {{ version }}
 
 source:
-  url: https://github.com/N3PDF/evolutionary_keras/archive/v{{ version }}.tar.gz
-  sha256: 39dc8ff742cea271b67605c029fb0ae66bbfa1cc2dd5574ee372d0ed81234a83
-
+  url: https://github.com/N3PDF/evolutionary_keras/archive/v.{{ version }}.tar.gz
+  sha256: 3f156dcf64448276a2745a39369f227166d75ed455218eb0e786be45f258065a
 build:
   number: 1
```
Comment (RoyStegeman) [inline recipe/meta.yaml]: The build number has to be set to 0 upon a new release

Comment (scarlehoff) [reply recipe/meta.yaml]: right, fixing

**Comment by scarlehoff** [comment]
This last commit builds fine locally. Some caveat is that I've had to set the channel priority to flexible so that the CI takes tensorflow from the default channel and not conda-forge, because conda-forge is still in 1.14 for some reason https://anaconda.org/conda-forge/tensorflow

**Comment by RoyStegeman** [comment]
This seems fine to me

**Pull request merged**